HOOKS.md

```diff
@@ -1,24 +1,24 @@
 # Hooks
 
-A hook is an executable file that Addon-operator executes when some event occurs. It can be a script or a compiled program written in any programming language.
+A hook is an executable file that the Addon-operator executes when some event occurs. It can be a script or a compiled program written in any programming language.
 
-Addon-operator pursues an agreement stating that the information is transferred to hooks via files and results of hooks execution are also stored in files. Paths to files are passed via environment variables. The output to stdout will be written to the log, except for the case with the configuration output (run with `--config` flag). Such an agreement simplifies the work with the input data and reporting the results of the hook execution.
+The Addon-operator pursues an agreement stating that the information is transferred to hooks via files and results of hooks execution are also stored in files. Paths to files are passed via environment variables. The output to stdout will be written to the log, except for the case with the configuration output (run with `--config` flag). Such an agreement simplifies the work with the input data and reporting the results of the hook execution.
 
 # Initialization of global hooks
 
-Global hooks can read and modify values in the global values storage which is available to all modules (see [VALUES](VALUES.md)). The global hook performs actions and discovers the values required by several modules.
+Global hooks can read and modify values in the storage of global values. Global values are also available to all modules (see [VALUES](VALUES.md)). The global hook performs actions and discovers the values required by several modules.
```

Comment:
```suggestion
Global hooks can read and modify values in the global values' storage. Global values are also available to all modules (see [VALUES](VALUES.md)). The global hook performs actions and discovers the values required by several modules.
```

Reply:
Storage is not significant here. This section is rewritten to explain 'global values changed' event.